Library/Homebrew/extend/pathname.rb

```diff
@@ -436,11 +436,12 @@ def env_script_all_files(dst, env)
   end
 
   # Writes an exec script that invokes a java jar
-  def write_jar_script(target_jar, script_name, java_opts = "")
+  def write_jar_script(target_jar, script_name, java_opts = "", java_version = "")
```

Comment:
`java_version: nil` and then you can specify it like this: `bin.write_jar_script libexec/"jenkins-cli.jar", "jenkins-lts-cli", java_version: "1.8"` rather than needing an empty `java_opts` argument.

Reply:
right, this is indeed better, thanks!